app/com/linkedin/drelephant/schedulers/AirflowScheduler.java

```diff
@@ -58,10 +62,14 @@ public AirflowScheduler(String appId, Properties properties, SchedulerConfigurat
   }
 
   private void loadInfo(String appId, Properties properties) {
-    // Update the 4 Ids
+    // examples:
+    // my_amazing_task_id
     _taskId = properties.getProperty(AIRFLOW_TASK_ID);
+    // 2016-06-27T01:30:00
```

Comment:
Can't there be multiple flows with the same date?

When we filter the db table by this field, it should return all the mapreduce jobs triggered by this flow.